server/integration/endpoint/src/main/resources/subsystem-templates/infinispan-endpoint.xml

```diff
@@ -135,6 +131,24 @@
          </router-connector>
       </replacement>
    </supplement>
+   <supplement name="memcached-local">
+      <replacement placeholder="@@default-cache-container@@" attributeValue="local" />
+      <replacement placeholder="CONNECTORS">
+         <memcached-connector socket-binding="memcached" cache-container="local" cache="default"/>
+      </replacement>
+   </supplement>
+   <supplement name="memcached-clustered">
+      <replacement placeholder="@@default-cache-container@@" attributeValue="clustered" />
+      <replacement placeholder="CONNECTORS">
+         <memcached-connector socket-binding="memcached" cache-container="clustered" cache="default"/>
+      </replacement>
+   </supplement>
+   <supplement name="websocket-local">
+      <replacement placeholder="@@default-cache-container@@" attributeValue="local" />
+      <replacement placeholder="CONNECTORS">
+         <websocket-connector socket-binding="websocket" cache-container="local" />
+      </replacement>
+   </supplement>
```

Comment:
Yes, that has no integration tests, and I think it is even less popular than memcached.